themes/default/content/docs/intro/concepts/plans.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Plans"
+meta_desc: Learn about Pulumi update plans and how they can be used.
+menu:
+  intro:
+    parent: concepts
+    weight: 10
+---
```

Comment:
To get the shorter URL, I think it's just a matter of adding an alias here, like we do here:

https://github.com/pulumi/pulumi-hugo/blob/d7a8112c4f57cf0990982061d28b0b0f7dafb1aa/themes/default/content/docs/get-started/_index.md?plain=1#L11-L15

```
aliases:
  - /updateplans/

```